main.nf

```diff
@@ -321,16 +338,19 @@ process fusion_inspector_preprocess {
 /*
  * Fusion Inspector
  */
+if (params.test) {
+    fusions = Channel.fromPath("$baseDir/tools/fusion-inspector/fusions.txt")
```

Comment:
Need to check if this work when executing:
```nextflow run nf-core/rnafusion ...```

Reply:
I'll run the pipeline one more time on UPPMAX but the pipeline worked as expected. I'll come back to this tomorrow after running the pipeline.

Reply:
Works as expected.